lineman/app/components/profile_page/profile_page.haml

```diff
@@ -1,21 +1,30 @@
 .loading-wrapper.container.main-container
   %loading{ng-if: '!profilePage.user'}
   %main.profile-page{ng-if: 'profilePage.user'}
-    %h4.lmo-h4{translate: 'profile_page.edit_profile'}
+    .profile-page-card
+      %h4.lmo-h4{translate: 'profile_page.edit_profile'}
 
-    %label{for: 'user-picture-field', translate: 'profile_page.picture_label'}
-    %user_avatar{user: 'profilePage.user'}
+      %label{for: 'user-picture-field', translate: 'profile_page.picture_label'}
+      %user_avatar{user: 'profilePage.user'}
 
-    %label{for: 'user-name-field', translate: 'profile_page.name_label'}
-    %input.form-control.profile-page__name-input{ng-model: 'profilePage.user.name'}
+      %label{for: 'user-name-field', translate: 'profile_page.name_label'}
+      %input.form-control.profile-page__name-input{ng-model: 'profilePage.user.name'}
 
-    %label{for: 'user-username-field', translate: 'profile_page.username_label'}
-    %input.form-control.profile-page__username-input{ng-model: 'profilePage.user.username'}
+      %label{for: 'user-username-field', translate: 'profile_page.username_label'}
+      %input.form-control.profile-page__username-input{ng-model: 'profilePage.user.username'}
 
-    %label{for: 'user-email-field', translate: 'profile_page.email_label'}
-    %input.form-control.profile-page__email-input{ng-model: 'profilePage.user.email'}
+      %label{for: 'user-email-field', translate: 'profile_page.email_label'}
+      %input.form-control.profile-page__email-input{ng-model: 'profilePage.user.email'}
 
-    %label{for: 'user-locale-field', translate: 'profile_page.locale_label'}
-    %select.form-control.profile-page__language-input{ng-model: 'profilePage.user.selectedLocale', ng-required: 'true', ng-options: 'locale.key as locale.name for locale in profilePage.availableLocales()', ng-change: 'setCurrentPrivacy()'}
+      %label{for: 'user-locale-field', translate: 'profile_page.locale_label'}
+      %select.form-control.profile-page__language-input{ng-model: 'profilePage.user.selectedLocale', ng-required: 'true', ng-options: 'locale.key as locale.name for locale in profilePage.availableLocales()', ng-change: 'setCurrentPrivacy()'}
 
-    %button.btn.btn-primary.profile-page__update-button{ng-click: 'profilePage.submit()', ng-disabled: 'isDisabled', translate: 'profile_page.update_profile'}
+      .profile-page__update-account
+        %button.btn.btn-primary.profile-page__update-button{ng-click: 'profilePage.submit()', ng-disabled: 'isDisabled', translate: 'profile_page.update_profile'}
+
+      .profile-page__change-password
+        %a{href: '', ng-click: 'profilePage.changePassword()', translate: 'profile_page.change_password_link'}
+    .profile-page-card{ng-if: 'profilePage.canDeactivateUser()'}
```

Comment:
TODO: The designs have some other crazy modal pop up letting you know why you can't deactivate, but I'm just hiding the card for now.